bin/MadGraph5_aMCatNLO/cards/production/13TeV/bbll_4FermionCI/bbll_4FermionCI_M-200To500_Lambda-1TeV_constLL_XQCUT-30/bbll_4FermionCI_M-200To500_Lambda-1TeV_constLL_XQCUT-30_proc_card.dat

```diff
@@ -0,0 +1,11 @@
+import model bbll_UFO
+define p = u d c s b u~ d~ c~ s~ b~ g
+define j = p
+generate p p > mu+ mu- NP^2>0 @0
```

Comment:
when you do np^2>0, will you get the first order that achieves this? so the pure interference term?

Reply:
What you get here is g * |NP X SM| + |NP X NP|.
So SM interference + pure NP terms.
And the g is either +4pi (constructive) or -4pi (destructive), if I understand your question correctly